tests/e2e/tests/stackdriver/adapter.yaml

```diff
@@ -0,0 +1,235 @@
+apiVersion: "config.istio.io/v1alpha2"
+kind: stackdriver
+metadata:
+  name: handler
+spec:
+  appCredentials: y
+  pushInterval: 5s
+  metricInfo:
+    server-request-count.metric.{{.Namespace}}:
+      kind: 3 # CUMULATIVE
+      value: 2 # INT64
+      metric_type: "istio.io/service/server/request_count"
+  logInfo:
+    server-accesslog-stackdriver.logentry.{{.Namespace}}:
+      labelNames:
+      - source_uid
+      - source_ip
+      - source_app
+      - source_principal
+      - source_name
+      - source_workload
+      - source_namespace
+      - source_owner
+      - destination_uid
+      - destination_app
+      - destination_ip
+      - destination_service_host
+      - destination_workload
+      - destination_name
+      - destination_namespace
+      - destination_owner
+      - destination_principal
+      - api_name
+      - api_version
+      - api_claims
+      - api_key
+      - request_operation
+      - protocol
+      - method
+      - url
+      - response_code
+      - response_size
+      - request_size
+      - request_id
+      - client_trace_id
+      - latency
+      - service_authentication_policy
+      - user_agent
+      - response_timestamp
+      - received_bytes
+      - sent_bytes
+      - referer
+  trace:
+    sampleProbability: 1.0
+---
+# apiVersion: "config.istio.io/v1alpha2"
```

Comment:
Was it your intent to leave this in here, but commented out?

Reply:
Yes, added comment